RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -336,6 +336,7 @@ namespace SDL {
                                                      float rt,
                                                      struct SDL::modules& modulesInGPU,
                                                      uint16_t& moduleIndex,
+                                                     const float& ptCut,
```

Comment:
pass by value should work

Reply:
I would interpret this comment to apply to all of the instances where pass by reference happens. Was that not the intention?

Reply:
Is passing by value preferred? I think in most areas of the code we pass things by reference.

Reply:
The compiler will probably optimize, since everything is inlined; if not passing, by ref needs more operations/memory.

Incidentally, in the same context, ` uint16_t& moduleIndex,` apparently can be const; the non-constness is more likely to prevent optimizations

my comment was for all cases

Reply:
Shall we then switch to pass by value for all the `ptCut` instances, so that we are consistent and doing the proper thing at least for this variable, since it is the subject of this PR?

Reply:
Resolved in my latest commit, rerunning the CI now.